x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/delete_domain_panel.tsx

```diff
@@ -27,6 +27,14 @@ export const DeleteDomainPanel: React.FC = ({}) => {
 
   return (
     <>
+      <EuiTitle size="s">
+        <h2>
+          {i18n.translate('xpack.enterpriseSearch.appSearch.crawler.deleteDomainPanel.title', {
+            defaultMessage: 'Delete domain',
+          })}
+        </h2>
+      </EuiTitle>
+      <EuiSpacer size="m" />
```

Comment:
Is this change supposed to be in this PR? Seems unrelated to the dedup feature.

Reply:
This component was the odd-man-out with its title in the container view compared to other already existing components and this new `DeduplicationPanel`.  So I moved this in for cosistency here, rather than do a whole separate PR for a few lines.